system-addon/content-src/components/CollapsibleSection/_CollapsibleSection.scss

```diff
@@ -73,7 +73,7 @@
   }
 
   &.active {
-    background: var(--newtab-element-active-color);
+    background: var(--newtab-element-hover-color);
```

Comment:
Somewhat confusing that `.active` uses `-hover-color`.. oh well

Reply:
we can rename the css class to 🤔 something.

Reply:
Eh, we can just leave it :p